### ISSUE TITLE
Is it possible to sync all repositories?

### ISSUE THREAD

**Issue by rc201612** [desktop/desktop#6482]
Hi, 

Is it possible to pull / sync all repositories? An icon shows me when a repository has changes to pull, but where there are many is there any way to do this in bulk?

Thanks,

**Comment by billygriffin** [comment]
Hi @rc201612, thanks for the issue! We've discussed supporting something like this but ultimately haven't found that it's a widespread enough need or causing a lot of friction for people (and there are a lot of pitfalls if something goes wrong). We responded to a similar request here: https://github.com/desktop/desktop/issues/2460#issuecomment-438813782

I'm curious what your workflow looks like that pulling individually is causing you to want something like this. Since we don't intend to support this anytime soon, I'm going to close this issue, but understanding a bit more about your workflow would be really helpful for us as we continue to evaluate potential things in the future.
